crates/oxc_linter/Cargo.toml

```diff
@@ -43,6 +43,7 @@ convert_case  = { workspace = true }
 language-tags = { workspace = true }
 mime_guess    = { workspace = true }
 url           = { workspace = true }
+unicode-segmentation = "1.11.0"
```

Comment:
Need call @Boshen for help, this crate is added for compute [`minimumDescriptionLength`](https://typescript-eslint.io/rules/ban-ts-comment#minimumdescriptionlength), how we deal with this?

Reply:
What's the problem with a normal `.len()`? I don't think it's a problem if there is 1 or 2 characters in difference.

Reply:
```ts
/**
 * @ts-expect-error 👨‍👩‍👧‍👦 */
```
If use the normal `.len()`, the length of `👨‍👩‍👧‍👦` is 27. And use `unicode_segmentation` to calcute the the length of `👨‍👩‍👧‍👦`, it would be 3.

![image](https://github.com/oxc-project/oxc/assets/22117876/7cc4c038-5c2f-4e39-8e46-3e6e041d3f4a)

So, If we use the normal `.len()` and description has emoji, These test cases about `minimumDescriptionLength` will be failed.

Reply:
Let's ignore this test for now, it's a rare case.

Reply:
> Let's ignore this test for now, it's a rare case.

ok